packages/salesforcedx-vscode-apex-debugger/src/utils/commandOutput.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export class CommandOutput {
+  private objectId: string;
+  private stdErr: string;
+  private stdOut: string;
+  private cmdMsg: string;
+  private cmdAction: string;
+
+  public saveId(id: string): void {
```

Comment:
Is there a reason for using `save` as the prefix instead of `set` as is usually done?